var/spack/repos/builtin/packages/scr/package.py

```diff
@@ -83,9 +85,9 @@ class Scr(CMakePackage):
             multi=False,
             description='File locking style for SCR.')
 
-    variant('cache_base', default='/tmp',
+    variant('cache_base', default='/dev/shm',
```

Comment:
Curious.  Is `/dev/shm` a directory that is expected to exist by default or is it created by the internal build process of the package?